js/components/scrubberBar.js

```diff
@@ -37,6 +37,27 @@ var ScrubberBar = React.createClass({
     this.scrubberBarWidth = this.getDOMNode().querySelector(".scrubberBar").clientWidth;
     this.scrubberBarHeight = this.getDOMNode().querySelector(".scrubberBar").clientHeight;
     this.playheadWidth = this.getDOMNode().querySelector(".playhead").clientWidth;
+
+    // Make sure component resize correctly after switch to fullscreen/inline screen
+    window.addEventListener('resize', this.handleResize);
+    window.addEventListener('webkitfullscreenchange', this.handleResize);
```

Comment:
There's a mixin for that - https://github.com/ooyala/html5-skin/blob/master/js/mixins/resizeMixin.js